public/orbs-public-beta-own-infra.md

```diff
@@ -0,0 +1,74 @@
+# Running Orbs v2 on your own infrastructure
+
+## Prerequisites
+
+* A Linux machine with Docker Swarm
+* Boyar v1.1.1 [binary download](https://s3.amazonaws.com/orbs-network-releases/infrastructure/boyar/boyar-v1.1.1.bin)
+* Connection to Ethereum node (`$ETHEREUM_ENDPOINT` url)
+* Directories `/var/efs`, `/var/efs/boyar-status` and `/var/efs/boyar-logs` should be created on the machine and have enough space (separate filesystem is recommended, you can use NFS if you like)
+
+## JSON configuration files
+
+### `mgmt.json`
+
+You need to replace `$ETHEREUM_ENDPOINT` with a URL of a real Ethereum node (you can use Infura free tier or your own node).
```

Comment:
can we use the same notation as the beta instructions page for fields the user need to change (<field name> + a legend with explanations so the names can be concise)? there were two reasons not to use the `$ETHEREUM_ENDPOINT` form for a field you need to replace manually - 
1. it suggests the use of env var which are not supported in our tools as far as i know...
2. in the beta instructions page the `polygon create` command contains a `$` as part of the concrete form to use which is confusing - it looks like something you should replace while you should not replace it..